mh_sha256/mh_sha256_block_avx512.asm

```diff
@@ -219,7 +219,7 @@ section .text
 	;; H becomes T2, then add T1 for A
 	;; D becomes D + T1 for E
 
-	vpaddd		T1, H, TMP3		; T1 = H + Kt
+	vpaddd		T1, H, [TBL + ((%%ROUND)*4)]{1to16}	; T1 = H + Kt
```

Comment:
Correct me if I'm wrong, but I think that we need this to make broadcast work, otherwise we'll end loading the wrong constants (after removing the duplicated values in the .data section).

Reply:
No worries :D